### PR TITLE
Conffile moves

### PR DESCRIPTION
**Pull request by rbalint**
Detect changes to moved conffiles and also not crash

### CODE REVIEW

File: test/test_conffile.py [hunk 1]
```diff
@@ -24,13 +24,33 @@ def setUp(self):
 No really.
 """)
 
+    def tearDown(self):
+        try:
```
Comment (mvo5) [inline test/test_conffile.py]: Could you use shutil.rmtree() here? That may simplify the code.

File: unattended-upgrade [hunk 2]
```diff
@@ -1098,14 +1118,21 @@ def conffile_prompt(destFile, prefix=""):
...
                 logging.debug("found conffile %s in new pkg but on dpkg "
                               "status" % conf_file)
                 pkg_md5sum = get_md5sum_for_file_in_deb(destFile, conf_file)
-                with open(prefix + conf_file, 'rb') as fp:
-                    if pkg_md5sum != apt_pkg.md5sum(fp):
-                        return True
+                if os.path.isdir(installed_conf_file):
+                    logging.debug("found conffile %s being a directory on the "
+                                  "system " % conf_file)
+                    pass
```
Comment (mvo5) [inline unattended-upgrade]: I don't think this `pass` here is needed.

File: test/test_conffile.py (hunk 1 above)
Comment (rbalint) [reply test/test_conffile.py]: Thanks, fixed that.

File: unattended-upgrade (hunk 2 above)
Comment (rbalint) [reply unattended-upgrade]: Good point, I felt that pass here helps following the logic, but it is indeed obsolete.

**Pull request merged**